internal/hmyapi/apiv1/backend.go

```diff
@@ -30,6 +30,8 @@ type Backend interface {
 	NetVersion() uint64
 	ProtocolVersion() int
 	ChainDb() ethdb.Database
+	SingleFlightRequest(key string, fn func() (interface{}, error)) (interface{}, error)
```

Comment:
dont think we should add single flight at this api level.

Reply:
why? unless deeper level is more beneficial.